code/integration-tests/src/androidTest/java/com/adobe/marketing/mobile/integration/identity/IdentityIntegrationTests.kt

```diff
@@ -463,6 +463,7 @@ class IdentityIntegrationTests {
         }
         MobileCore.resetIdentities()
         countDownLatchSecondNetworkMonitor.await()
+        Thread.sleep(20)
```

Comment:
What is the need for such a long sleep here? Can you reduce the timeout?

Reply:
`mid` will be refreshed in the local storage after the network call, So just wait a bit here to get the refreshed `mid` value.